metricbeat/module/docker/network/data.go

```diff
@@ -5,32 +5,70 @@ import (
 	"github.com/elastic/beats/metricbeat/mb"
 )
 
-func eventsMapping(netsStatsList []NetStats) []common.MapStr {
-	myEvents := []common.MapStr{}
+func eventsMapping(r mb.ReporterV2, netsStatsList []NetStats) {
 	for _, netsStats := range netsStatsList {
-		myEvents = append(myEvents, eventMapping(&netsStats))
+		eventMapping(r, &netsStats)
 	}
-	return myEvents
 }
 
-func eventMapping(stats *NetStats) common.MapStr {
-	event := common.MapStr{
-		mb.ModuleDataKey: common.MapStr{
-			"container": stats.Container.ToMapStr(),
+func eventMapping(r mb.ReporterV2, stats *NetStats) {
+	container := stats.Container.ToMapStr()
+	container.Put("runtime", "docker")
+
+	// Inbound traffic event
+	r.Event(mb.Event{
+		RootFields: common.MapStr{
+			"container": container,
```

Comment:
Not sure I would mix the `container` to the root level into this PR.

Reply:
The idea is to move the `container` fields to ECS structure too.